samples/RazorLight.Samples/Samples.EntityFrameworkProject.csproj

```diff
@@ -1,29 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
-    <PreserveCompilationContext>true</PreserveCompilationContext>
-	<IsPackable>false</IsPackable>
-  </PropertyGroup>
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFrameworks>netcoreapp3.0;netcoreapp3.1;net5.0</TargetFrameworks>
+        <PreserveCompilationContext>true</PreserveCompilationContext>
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
```

Comment:
Dont remove as this is needed to help guide the .net dependency resolution engine towards the right dependencies to use.